web/satellite/src/components/account/billing/BillingArea.vue

```diff
@@ -1,9 +1,9 @@
-// Copyright (C) 2019 Storj Labs, Inc.
+// Copyright (C) 2022 Storj Labs, Inc.
 // See LICENSE for copying information.
 
 <template>
     <div class="account-billing-area">
-        <div v-if="isNewBillingScreen">
+        <div v-if="!isNewBillingScreen">
```

Comment:
you will need to change this flag back before I will approve. Right now, the new billing screen is shown when the flag is turned off
It's fine if you flip it like this to do local testing, but if you commit that change, I can't approve even if everything else looks good. I would suggest flipping the flag back and adding it to a commit. Then, when testing locally just switch it back to `!isNewBillingScreen`, but don't add that change to a future commit. That way, what is pushed on Github always has the expected flag behavior